zip_wheels.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/zsh
+
+set -e
+
+OUTPUT_DIRECTORY="./src/connectors/snowflake/trulens/data/snowflake_stage_zips"
```

Comment:
It's the latter. I've added a comment in the file now. The reason is because it needs to be in a (sub)directory of `/src/connectors/snowflake` because that's the wheel that'll actually house the data. It also should be in `trulens` since that's where everything is. So I just made a `data/snowflake_stage_zips` directory in there.